### PR TITLE
Update Blazars version of singularity

### PR DESCRIPTION
**Pull request by Jw0x47**
Update singularity to latest version, we're very behind.

### CODE REVIEW

**Comment by Jw0x47** [comment]
This has been Merged to staging.

File: pom.xml [hunk 1]
```diff
@@ -267,6 +267,23 @@
   <build>
     <pluginManagement>
       <plugins>
+        <!-- This lets us use logback instead of simplelogger with maven -->
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-checkstyle-plugin</artifactId>
+          <dependencies>
+            <dependency>
```
Comment (gchomatas) [inline pom.xml:275]: These dependencies do not seem to relate to logback. Are you sure we need them?

**Comment by gchomatas** [comment]
🚢

File: pom.xml (hunk 1 above)
Comment (Jw0x47) [reply pom.xml:275]: This comment explains why they are necessary & related.
https://github.com/jcgay/maven-color/issues/11#issuecomment-125326792

**Pull request merged**